core/src/test/java/org/elasticsearch/index/query/MissingQueryBuilderTest.java

```diff
@@ -68,96 +63,8 @@ private String getRandomFieldName() {
     }
 
     @Override
-    protected Query doCreateExpectedQuery(MissingQueryBuilder queryBuilder, QueryParseContext context) throws IOException {
-        final boolean existence = queryBuilder.existence();
-        final boolean nullValue = queryBuilder.nullValue();
-        String fieldPattern = queryBuilder.fieldPattern();
+    protected void doAssertLuceneQuery(MissingQueryBuilder queryBuilder, Query query, QueryParseContext context) throws IOException {
```

Comment:
We can leave it then, maybe add some note to revisit it later once again to see if we can extend test a bit here?